tasks/main.yml

```diff
@@ -11,55 +11,57 @@
         - "{{ role_path }}/vars"
 
 - name: Install system packages necessary for the tasks to follow
-  package:
+  ansible.builtin.package:
     name: "{{ package_names }}"
 
 - name: Copy user-mode systemd service file for creating file share symlink
-  copy:
+  ansible.builtin.copy:
     dest: /etc/systemd/user/
     mode: 0644
     src: create-fileshare-symlink.service
 
 - name: Install COOL background image
   block:
     - name: Create COOL backgrounds directory
-      file:
+      ansible.builtin.file:
         mode: 0755
         path: /usr/share/backgrounds/cool
         state: directory
 
     - name: Install COOL background images from cool-system GitHub repository
-      unarchive:
+      ansible.builtin.unarchive:
         creates: /usr/share/backgrounds/cool/cisa_cool_retro_0.png
         dest: /usr/share/backgrounds/cool
         extra_opts:
           - --strip-components=3
+          - --wildcards
+          - "*/assets/desktops"
         remote_src: yes
         src: https://github.com/cisagov/cool-system/archive/develop.tar.gz
```

Comment:
I know this works, but would it be better to use `include` to extract the file(s) we want instead of relying on arguments for the underlying tool being run by the module?

Reply:
Excellent point!  See commit 74f4442.